tests/ttnn/unit_tests/operations/test_maxpool2d.py

```diff
@@ -113,7 +113,7 @@ def run_max_pool(
     # interleaved_mem_config = ttnn.L1_MEMORY_CONFIG
     # output = ttnn.to_memory_config(output, interleaved_mem_config)
     output_host = output.cpu()
-    output_pytorch_padded = ttnn.to_torch(output_host)
+    output_pytorch_padded = torch.Tensor(ttnn.to_torch(output_host))
```

Comment:
why is this needed? ttnn.to_torch returns torch.Tensor

Reply:
It doesn't really return pure torch tensor -- needs ttnn to subsequently read/manipulate it. This was the recommended way to convert it into pure torch.

Reply:
Yeah can confirm. If you don't do it torch function will crash with segmentation fault.

Reply:
🫠 okay, good to know